nano-sync/src/circuits/mnt4/pk_tree_1.rs

```diff
@@ -0,0 +1,436 @@
+use std::fs::File;
```

Comment:
Not quite. pk_tree 1 and 3 are indeed equal. And pk_tree 0, 2 and 4 are also equal. I think I had them all different at the time because I thought I was going to hardcode the vk in them, not feeding them through a file.
But I'm not sure how to have the filename for the vk as a parameter. I can't have it as a parameter to `new()`, that function only returns a PKTreeCircuit struct and all the fields of that struct must be public/private inputs to the circuit.
One option might be adding it as a parameter to the generate_constraints function?

Reply:
Ah, I compared a few files in my editor and probably compared 1 with 3 and 2 with 4. 👍 
> I can't have it as a parameter to `new()`, that function only returns a PKTreeCircuit struct and all the fields of that struct must be public/private inputs to the circuit.

I don't get why you cannot have it as a parameter to `new()`.

Reply:
Because whatever parameter I add to 'new()' ZEXE will assume that it is an input to the ZK circuit.

Reply:
Done!